### DISCUSSION TITLE
Reversing sweeps in radar

### DISCUSSION THREAD

**Discussion by wxwolff** [ARM-DOE/pyart#1483] [Q&A] [answered]
I have an ODIMH5 file that when read in with pyart.aux_io.read_odim_h5 provides a radar object with the highest elevation is sweep #0 and the lowest is sweep # (radar.nsweeps-1).  Is there a way to reverse this and maintain all of of the radar info?

**Comment by mgrover1** [comment, marked as answer]
Sorry for the very long delay here - when using the `pyart.util.subset_radar` function, it will sort the radar when processing your subset. You could pass in all of your fields, and specify from 0 deg to 90 deg elevations to include the full radar data?

```python
radar = pyart.aux_io.read_odim_h5("...")
radar = pyart.util.subset_radar(radar, list(radar.fields), ele_min=0., ele_max=90.)
```

This would return the same radar object, with the elevations sorted.
